tests/integration/nvidia_tuned/test_host_setup.py

```
@@ -0,0 +1,590 @@
+
+def test_wait_rdma_vfs_falls_back_on_a_bad_poll_interval(node):
+    """POLL_INTERVAL_SECS=0 would stop elapsed advancing and busy-spin until systemd."""
+    stage_netdevs(node, vfs=0)
+
+    rc = wait_helper(node, EXPECTED_VFS=4, TIMEOUT_SECS=4, POLL_INTERVAL_SECS=0)
+    assert rc == 0, output(node)
+    assert said(node, "POLL_INTERVAL_SECS='0' is not a positive integer"), output(node)
+
+
+@pytest.mark.parametrize(
+    "name,value",
```

Ruff (0.16.1)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)